r-pkg/R/es_search.R

```diff
@@ -112,7 +119,6 @@ es_search <- function(es_host
         , assertthat::is.string(scroll)
         , scroll != ""
         , max_hits >= 0
-        , assertthat::is.count(n_cores)
```

Comment:
this should actually still be here @drkarthi . It's a guarantee that _as of this point in the code_ `n_cores` is a count. Even though that `if` block above would make this always TRUE, I don't consider it dead or wasted code to have this assertion here.

Can you please reinstate it?